docs/fedora-livemedia.ks

```diff
@@ -389,4 +388,13 @@ syslinux
 -gfs2-utils
 -reiserfs-utils
 
+# This package is needed to boot the iso on UEFI
+shim
+shim-ia32
+grub2
```

Comment:
This should be `grub2-pc`?

Reply:
I'm not sure, grub2 works fine in my tests.

Reply:
fedora docs suggest "dnf install grub2" which appears to be a magic package that selected grub2-pc on my f29